src/osmf/lib/HDSPlayer.as

```diff
@@ -430,6 +432,7 @@ package
       if (_playerState == MediaPlayerState.READY || _playerState == MediaPlayerState.PAUSED 
           || _playerState == MediaPlayerState.BUFFERING)
       {
+        _playCompleted = false;
```

Comment:
Hi Sarita,
The thing is that the variables need to be correct whether or not we are using them at this time.  I could easily change the code to call getCurrentTime at other times and it should always work; otherwise we are introducing bugs that are not yet exposed, which could become exposed later.  The ask is that if the stream url is changed, we just set _playCompleted to false because at that point the play is not compelted